js/views-manager.js

```diff
@@ -1,13 +1,15 @@
-// TODO:FIXME: 
```

Comment:
@acabunoc I would suggest to check out changes in this file commit by commit.

In https://github.com/mozilla/network-pulse/pull/164/commits/ea44a25547af87dcf4b8ab5ef8637139107bc21b I changed the way we render/display project cards. Previously we removed/added `<div class="project" ..>` into DOM every time we switch tab/page. Now on Featured / Latest / Issues Tabs project cards' visibility are toggled instead. Navigate to / from the Favs Tab is different and requires remove & re-rendering project cards into DOM since faved projects are ordered by the time projects have been faved (instead of the timestamp of the Google Form submission)